src/lib/sfge/AbstractSfgeEngine.ts

```diff
@@ -260,17 +294,25 @@ export abstract class AbstractSfgeEngine extends AbstractRuleEngine {
 		}
 	}
 
-	protected processStdout(output: string): RuleResult[] {
-		// Pull the violation objects from the output.
+	/**
+	 * Seeks and returns violations within {@code output}, returning an empty array if none can be found.
+	 * @param output String containing (possibly among other things) the {@code stdout} from a transaction.
+	 * @protected
+	 */
+	protected parseViolations(output: string): RuleResult[] {
```

Comment:
`processStdout` was a misleading name for this method, since a failure state means `output` contains both `stdout` _and_ `stderr`. So I renamed it to more accurately describe the specific functionality of pulling violations out of whatever string it's given.